doc/man7/EVP_PKEY-EC.pod

```diff
@@ -47,10 +48,67 @@ An B<EVP_PKEY> context can be obtained by calling:
     EVP_PKEY_CTX *pctx =
         EVP_PKEY_CTX_new_from_name(NULL, "EC", NULL);
 
+An B<EVP_PKEY> ECDSA or ECDH key can be generated with a "P-256" named group by
+calling:
+
+    EVP_PKEY *key = NULL;
+    OSSL_PARAM params[2];
+    EVP_PKEY_CTX *gctx =
+        EVP_PKEY_CTX_new_from_name(NULL, "EC", NULL);
+
+    EVP_PKEY_keygen_init(gctx);
+
+    params[0] = OSSL_PARAM_construct_utf8_string(OSSL_PKEY_PARAM_EC_NAME,
+                                                 "P-256", 0);
+    params[1] = OSSL_PARAM_construct_end();
+    EVP_PKEY_CTX_set_params(gctx, params);
+
+    EVP_PKEY_gen(gctx, &key);
+
+    EVP_PKEY_print_private(bio_out, key, 0, NULL);
+    ...
+    EVP_PKEY_free(key);
+    EVP_PKEY_CTX_free(gctx);
+
+An B<EVP_PKEY> ECCDH key can be generated with a "K-571" named group by calling:
```

Comment:
"ECDH" should be spellt "ECDH" :wink:

Reply:
For this example it is actually ECCDH not ECDH - The extra C is for the cofactor.

Reply:
Hmm maybe I am confused.. Looks like that c means computational.

What I am trying to refer to is that this example uses the 
Elliptic   Curve   Cryptography   Cofactor   Diffie-Hellman   (ECC   CDH) Primitive
As defined in https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-56Ar3.pdf Section 5.7.1.2

Reply:
Gotcha...  although, "ECCDH" mostly looks like a kind of in-between, and doesn't "click" with me.  Maybe it would be better to actually write `ECC CDH`...

Reply:
I think "ECC DH" is more clear.  And "spellt" should be spelled like "spelt" (sic)

Reply:
> I think "ECC DH" is more clear.

In what way?

> And "spellt" should be spelled like "spelt" (sic)

Ah!  Thanks

Reply:
In fact, I think it should be "Elliptic Curve Cofactor DH" since ECDH is common, but the openssl community is probably not used to cofactor.

Reply:
Yeah, but this is confusing, as ECCDH could match Elliptic Curve Cryptography Diffie-Hellman just as much as Elliptic Curve Cofactor Diffie-Hellman.  And especially the separate "ECC" is, as far as I know, much more commonly known to mean Elliptic Curve Cryptography.  We *might* get away with "EC CDH", if you really want to get rid of one C...  after all, the cofactor thing is much more tied to the DH aspect (isn't it?)...

Reply:
will change to EC CDH (Cofactor Diffie-Hellman)